doc/ReleaseNotes.md

```diff
@@ -13,29 +13,171 @@ r151034 release repository: https://pkg.omniosce.org/r151034/core
 
 ## Upgrade Notes
 
+It is only possible to upgrade to this release from r151030 or r151032. If
+starting from an earlier release, upgrade in stages following the guide at
+https://omniosce.org/upgrade
+
 ## New features since r151032
 
 ### System Features
 
+* OmniOS now supports running an NFS server in a zone. NFS sharing can be
+  enabled by setting the `sharenfs` property on a dataset just as in the
+  global zone.
+
```

Comment:
Native zone.  Can't do it (easily, if at all) in LX.  Also, do any of your newer native brands do what SmartOS does and not give root@zone its own dataset/mountpoint?  If so, then you can't share NFS on such a dataset.  If you can `zfs set` in the zone, you can share it, but some old-timers might use share(1M) or sharemgr(1M) and if they share a directory on a filesystem the zone doesn't own, they'll get EPERM or something else.

Reply:
Thanks.
No, none of our brands do that apart from kvm/bhyve.
This should be usable in ipkg, lipkg, sparse or pkgsrc. I'll make the note more explicit.

Reply:
So if those zone brands have $ZONEROOT owned by root@zone, which you just said they do, you can ALSO use the share(1M)/share_nfs(1M) command on any directory in-zone for finer-grained directory access.